src/main/webapp/view/discussion/discussion.js

```diff
@@ -12,26 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import DiscussionArea from './discussion-area.js';
+import View from '../view.js';
 
 export const COMMENT_TYPE_REPLY = 'REPLY';
 export const COMMENT_TYPE_QUESTION = 'QUESTION';
 export const COMMENT_TYPE_NOTE = 'NOTE';
 
-export let /** DiscussionArea */ discussion;
-
-/**
- * Loads the lecture discussion.
- */
-export async function intializeDiscussion() {
-  discussion = new DiscussionArea(window.LECTURE);
-  discussion.initialize();
-  // This is used as the `onclick` handler of the new comment area submit
-  // button. It must be set after discussion is initialized.
-  window.postNewComment = discussion.postNewComment.bind(discussion);
-}
-
 /** Seeks discussion to `timeMs`. */
 export function seekDiscussion(timeMs) {
```

Comment:
can we delete this?  or are we still waiting on event controller stuff

Reply:
the `seekDiscussion` function

Reply:
waiting on the event controller stuff